.travis.yml

```diff
@@ -16,13 +16,15 @@ before_install:
 - echo ${DOCKER_PASSWORD} | docker login -u=decaftravis --password-stdin
 install:
 - docker build -t ${IMAGE}:alpine -t ${IMAGE}:alpine-${IMAGE_HASH_TAG} ./alpine
+- docker build -t ${IMAGE}:alpine-compiler ./alpine-compiler
 - docker build -t ${IMAGE}:debian -t ${IMAGE}:debian-${IMAGE_HASH_TAG} ./debian
 script:
 - docker run --rm ${IMAGE}:alpine gunicorn --version
 - docker run --rm ${IMAGE}:debian gunicorn --version
 after_success:
 - docker push ${IMAGE}:alpine
 - docker push ${IMAGE}:alpine-${IMAGE_HASH_TAG}
+- docker push ${IMAGE}:alpine-compiler
```

Comment:
I would also create a tag and push `${IMAGE}:alpine-compiler-${IMAGE_HASH_TAG}`.